doc/ExampleApplications.xml

```diff
@@ -104,9 +104,9 @@ gap&gt; EulerCharacteristic(newSurf);
         be represented as a triangulation of the sphere. We might want to
         know how many vertices there are of different degrees. This
         can be found out by using <K>VertexCounter</K> 
```

Comment:
Shouldn't this also say CounterOfVertices here? There is another occurrence of VertexCounter in the manual beside this. I am not sure if this is intended :)